### PR TITLE
Migrate from Rubocop to Standard

### PR DESCRIPTION
**Pull request by markets**
`StandardRB` Ruby Style Guide, with linter & automatic code fixer

**WHY?**
https://github.com/testdouble/standard#why-should-i-use-ruby-standard-style

**EXTRAS** 
- Migrate from Hound to GH Actions

### CODE REVIEW

**Comment by markets** [comment]
@thibaudgg not sure why the `amoeba/standardrb-action` failed 🤔, StandardRB Linter passes for me locally ✅ 

So for now, I just added the linter as an extra step calling manually `bundle exec standardrb`. We won't have comments reporting in pull requests, but this is fine for me too 😃 we can integrate an action later in case we miss them?

**Comment by thibaudgg** [comment]
@markets I would try to fix it, it's nice to have the comments directly in the pull requests. 

Did you already try to fix the permissions? https://github.com/marketplace/actions/standardrb#common-issues

**Comment by markets** [comment]
Thanks @thibaudgg! Defining the permissions was the trick:

```
permissions:
  checks: write
  contents: read
```

File: README.md [hunk 1]
```diff
@@ -175,16 +179,21 @@ VideoInfo.disable_providers = [] # enable all providers
...
 
-## Maintainers
+### Maintainers
 
 - [Thibaud Guillaume-Gentil](https://github.com/thibaudgg) ([@thibaudgg](https://twitter.com/thibaudgg))
-- [Vincent Heuken](https://github.com/vheuken) ([@vheuken](https://github.com/vheuken))
-- [João Vieira](https://github.com/joaocv3) ([@joaocv3](https://github.com/joaocv3))
+- [Vincent Heuken](https://github.com/vheuken)
+- [João Vieira](https://github.com/joaocv3)
+- [Marc Anguera](https://github.com/markets)
```
Comment (thibaudgg) [inline README.md:195]: Welcome! 🙌🏻

**Review by thibaudgg — APPROVED** [review]
👍🏻

**Pull request merged**